tests/schema.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
Just for consistency.
```suggestion
<?php

```